modules/nw-configure-ingress-access-logging.adoc

```diff
@@ -116,3 +116,47 @@ spec:
   logging:
     access: null
 ----
+
+Allowing IngressController to Modify the HAProxy Log Length when using a Sidecar
+
+* Configure `httpLogFormat` within IngressController. Leaving it blank will default to standard http log format. 
+Use `spec.logging.access.destination.syslog.maxLength` if you're using `spec.logging.access.destination.type: Syslog`.
```

Comment:
A couple points:
1. This says to configure the `httpLogFormat` parameter, but does not show that in the YAML. Should it be included?
2. Either "the `IngressController` resource" (labelled resource name) or "the Ingress Controller" (plain text)
3. Generally avoid contractions, so sub "you are" in for "you're" (here and in the line later on)

Reply:
I think the two sentences, "Configure `httpLogFormat` within IngressController. Leaving it blank will default to standard http log format. " can be omitted from the list item about configuring maximum log length.  `httpLogFormat` and `maxLength` are separate API fields.